File: tkpdstream/src/main/java/com/tokopedia/tkpdstream/chatroom/domain/pojo/Statistic.java

package com.tokopedia.tkpdstream.chatroom.domain.pojo;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Statistic {

    @SerializedName("total_voter")
    @Expose
    private int totalVoter;
    @SerializedName("statistic_options")
    @Expose
    private List<StatisticOption> statisticOptions = null;

    public int getTotalVoter() {
        return totalVoter;
    }

    public void setTotalVoter(int totalVoter) {
        this.totalVoter = totalVoter;
    }

    public List<StatisticOption> getStatisticOptions() {
        return statisticOptions;
    }

    public void setStatisticOptions(List<StatisticOption> statisticOptions) {
        this.statisticOptions = statisticOptions;
    }

}

File: features/user/groupchat/src/main/java/com/tokopedia/groupchat/chatroom/domain/pojo/poll/Statistic.kt
package com.tokopedia.groupchat.chatroom.domain.pojo.poll

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Statistic {

    @SerializedName("total_voter")
    @Expose
    var totalVoter: Int = 0
    @SerializedName("statistic_options")
    @Expose
    var statisticOptions: List<StatisticOption> = ArrayList()

}
